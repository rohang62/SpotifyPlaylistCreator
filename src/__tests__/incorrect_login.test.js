import React from "react";
import renderer from "react-test-renderer";
import IncorrectLogin from "./../incorrect_login";

test("renders incorrect login screen", () => {
	let state = { username : '', password : '' };
	const Tree = renderer.create(<IncorrectLogin state = {state}/>).toJSON();
	expect(Tree).toMatchSnapshot();
});
