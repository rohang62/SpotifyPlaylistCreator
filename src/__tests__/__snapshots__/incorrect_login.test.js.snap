// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders incorrect login screen 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="spotify_logo.svg"
    />
    <div>
      <label>
        Username:
        <input
          onChange={[Function]}
          type="text"
          value=""
        />
      </label>
    </div>
    <div>
      <label>
        Password:
        <input
          onChange={[Function]}
          type="password"
          value=""
        />
      </label>
    </div>
    <br />
    <div
      className="Incorrect"
    >
      Incorrect details. Please try again.
    </div>
    <button
      className="button"
      onClick={[Function]}
    >
       Submit 
    </button>
  </header>
</div>
`;
